src/main/java/com/spartronics4915/frc2020/commands/SpinToColorCommand.java

```diff
@@ -23,8 +23,7 @@ public void initialize() {
     public void execute() {
         String targetColor = mPanelRotator.getTargetColor();
         targetColor = DriverStation.getInstance().getGameSpecificMessage();
-        switch (targetColor.charAt(0))
-        {
+        switch (targetColor.charAt(0)) {
           case 'B' :
```

Comment:
Right now this is a lot of redundant code - we can discuss the simplest / best way to implement color switching tonight...

Reply:
@j-james I've made this way way smaller; that was just copied from the example code, which was way too complicated. I will push in a few minutes with a much much smaller version.

### Also:
**This needs something to stop the command if there is no color value**—if they accidentally hit the button but there is no color data, this should automatically leave. I tried making it so that if targetColor is not one of the possible values it becomes "error" which makes isFinished return true, but there was a problem I don't understand with trying to use that variable there.